docker/build/installers/install_qa_tools.sh

```diff
@@ -44,6 +44,13 @@ apt-get -y update && \
 
 bash /tmp/installers/install_gperftools.sh
 
+bash /tmp/installers/install_benchmark.sh
+
+# TechDoc generation
+bash /tmp/installers/install_doxygen.sh
```

Comment:
How about simply "apt install doxygen"?

Reply:
> How about simply "apt install doxygen"?
Both are OK.  Doxygen installed via apt depends on  libclang1-6.0, and we have llvm-clang 10.0 installed already, to avoid different versions of clang stuff,  I chose to install doxygen from source. However, this approach has the disadvantage of less maintainability.